<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addForm</class>
 <widget class="QWidget" name="addForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <widget class="QTextEdit" name="messageTextbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTextbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sernameTextbox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTextbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passTextbox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adressTextbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Адрес</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firmTextbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Фирма счетчика</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelTextbox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Модель счетчика</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idTextbox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ID счетчика</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageHint">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сообщение</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
